src/include/brain/brain_defs.h

```diff
@@ -5,36 +5,136 @@ namespace terrier::brain {
 enum class ExecutionOperatingUnitType : uint32_t {
   INVALID,
 
+  /**
+   * AGGREGATE_BUILD
+   * num_rows: # input rows to aggregation
+   * cardinality: # unique values (although probably more correct is # of unique values based on group by clause)
+   */
   AGGREGATE_BUILD,
+  /**
+   * AGGREGATE_ITERATE
+   * num_rows: # rows output by aggregation  // TODO(WAN): when is this not 1?
```

Comment:
Is this TODO still current?

Reply:
Yes; an aggregation typically smashes a whole bunch of values into a single value right? When will there be more than 1 value in an aggregation's output? I cannot think of any right now.

Reply:
Well in the case of a group by there will be more values (same number of rows as the distinct values in the group by column).

Reply:
Got it.